apps/aecore/lib/aecore/naming/tx/name_revoke_tx.ex

```diff
@@ -41,8 +42,27 @@ defmodule Aecore.Naming.Tx.NameRevokeTx do
   # Callbacks
 
   @spec init(payload()) :: t()
+  # TODO integrate Identifiers
```

Comment:
is this todo still to be done?

Reply:
This is just a part of unused comments, will remove them soon.